### DISCUSSION TITLE
List the Benefits of Payjoin

### DISCUSSION THREAD

**Discussion by DanGould** [payjoin/rust-payjoin#91] [General]
- fee savings via opportune consolidation 
- fee savings via [transaction cut-through](https://bitcointalk.org/index.php?topic=281848.0) for batches
- time savings via transaction cut-through by settling many transactions in one rather than waiting for confirmations
- lightning channel savings via transaction cut-through [lightning payjoin](https://chaincase.app/words/lightning-payjoin)
- break third party surveillance common input ownership heuristic effectiveness
- address re-use prevention through interaction
- compliant with existing legal regimes (IANAL)

**Comment by robbaltzer** [comment]
breaks change address heuristic too, no?

**Reply by ghost** [reply]
That is mentioned in BIP 78
